core/src/client.rs

```diff
@@ -293,11 +293,16 @@ impl APIClient {
         }
     }
 
-    pub async fn query_wait(&self, sql: &str) -> Result<QueryResponse> {
-        let resp = self.query(sql).await?;
+    pub async fn query(&self, sql: &str) -> Result<QueryResponse> {
+        let resp = self.start_query(sql).await?;
         self.wait_for_query(resp).await
     }
 
+    // TODO(liyz): for backward compatibility, remove it later when we've make sure the derived drivers did not call this method.
```

Comment:
This maybe not necessary, since we are releasing `core` and `driver` to the same version.

Reply:
thank you for reminder, i'll remove it